stdlib/Logging/src/Logging.jl

```diff
@@ -54,6 +54,18 @@ const Warn = Base.CoreLogging.Warn
 Alias for [`LogLevel(2000)`](@ref LogLevel).
 """
 const Error = Base.CoreLogging.Error
+"""
+    BelowMinLevel
+
+Alias for [`LogLevel(-1_000_001)`](@ref LogLevel).
```

Comment:
imo it would be helpful to have a more descriptive docstring; e.g. what are these for? how should they be used?

Reply:
Yeah, I think that would be good. I'm not sure what the intended usage is. I've been using `BelowMinLevel` in `Logging.disable_logging(Logging.BelowMinLevel)`, which I use as a workaround for #52234.

@c42f Could you weigh in on the intended usage of `BelowMinLevel` and `AboveMaxLevel`?

Reply:
It looks like none of the others have a useful description either, so this seems to fit in